.github/workflows/triage.yml

```diff
@@ -10,6 +10,10 @@ on:
     # https://github.com/actions/labeler/issues/12
     - cron: '*/5 * * * *'
 
+concurrency:
+  group: issue-triage
+  cancel-in-progress: true
```

Comment:
It is sort of unfortunate that we have to cancel in-progress jobs rather than cancel ourselves since there is already a job in progress :-/